src/routes/project/[projectId]/dashboard/+page.ts

```diff
@@ -1,38 +1,64 @@
 import { backendService } from "$lib/grpc-api";
 import type { PageLoad } from "./$types";
-import { type Project, type Project_Paper } from "$lib/model/api/project";
+import {
+    PaperDecision,
+    type Project,
+    type Project_Information_DecisionStatistics,
+} from "$lib/model/api/project";
 import type {
     PaperListEntryInterface,
     ProjectInformationInterface,
     StageProgressInterface,
 } from "$lib/model/component-interfaces";
-import { getStatusText, groupBy } from "$lib/utils/common-helper";
+import type { Timestamp } from "$lib/model/api/google/protobuf/timestamp";
 
-async function groupProjectPapersInStageByDecision(
-    project: Project,
-): Promise<{ [key: string]: Project_Paper[] }> {
-    const papersOfStage = await backendService
-        .getAllProjectPapersForProject({ id: project.id })
-        .response.then((papers) =>
-            papers.projectPapers.filter(
-                (projectPaper) => projectPaper.stage === project.currentStage,
-            ),
-        );
+/**
+ * Parses a Timestamp object into a date object.
+ *
+ * @param timestamp - The timestamp given as object of seconds and nanoseconds relative to an epoch of UTC
+ * @param errorMessage - The message in the error thrown when the parsing is not possible
+ */
+function parseTimestamp(timestamp?: Timestamp, errorMessage: string = ""): Date {
+    if (!timestamp) {
+        throw new Error(errorMessage, { cause: "NoDate" });
+    }
 
-    return groupBy(papersOfStage, getStatusText);
+    return new Date(Number(timestamp.seconds) * 1000);
 }
 
-async function requestProjectInformation(project: Project): Promise<ProjectInformationInterface> {
-    // TODO: add a real call to the backend (extend API + extend mock backend), waiting for #7 in snowballr-api
-    const startDate = new Date();
-    startDate.setDate(startDate.getDate() - 5);
-    const daysInStage = 2;
+async function requestProjectInformation(
+    project: Project,
+    decisionStatistics: Project_Information_DecisionStatistics,
+): Promise<ProjectInformationInterface> {
+    const projectInformation = await backendService.getProjectInformation({ projectId: project.id })
+        .response;
+    if (!projectInformation.creationDate) {
+        throw new Error("Couldn't load start date of project.", {
+            cause: "NoDate",
+        });
+    }
+    const startDate = parseTimestamp(
+        projectInformation.creationDate,
+        "Couldn't load start date of project.",
+    );
```

Comment:
Now that we use the parsing method, we don't require the check.
```suggestion
    const projectInformation = await backendService.getProjectInformation({ projectId: project.id })
        .response;
    const startDate = parseTimestamp(
        projectInformation.creationDate,
        "Couldn't load start date of project.",
    );
```

Reply:
I am so lost, i forgot to delete it.